sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1627,8 +1568,9 @@
       </build>
     </profile>
 
+    <!-- TODO (JG): Is this profile still being used? -->
```

Comment:
Yes, this is needed to add that additional system property to the build to allow libraries using SecurityManager to build in Java 18+

Reply:
Will leave it in - thanks.